mlir/lib/Target/LLVM/NVVM/Target.cpp

```diff
@@ -220,6 +221,16 @@ class NVPTXSerializer : public SerializeGPUModuleBase {
   std::optional<SmallVector<char, 0>>
   moduleToObject(llvm::Module &llvmModule) override;
 
+  /// Get LLVMIR->ISA performance result.
+  /// Return nullopt if moduleToObject has not been called or the target format
+  /// is LLVMIR.
+  std::optional<int64_t> getLLVMIRToISAPerfResult();
```

Comment:
```suggestion
  std::optional<int64_t> getLLVMIRToISATimeInMs();
```

Just a nit: making the name more explicit (I'm assuming the result is in `ms` but please adjust the unit).

(same below)

Reply:
Have modified, thx.

Reply:
Could you help merge it?